Gemfile

```diff
@@ -109,7 +109,7 @@ gem 'statsd-instrument', '~> 2.6.0' # versions beyond 2.6 deprecate config and c
 gem 'swagger-blocks'
 gem 'typhoeus'
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
+gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: '10-10cg-insurance-questions'
```

Comment:
Do we want to wait until the other PR is merged, before merging this change?

Reply:
Yes, waiting for the schema to be merged and I can update this to reference master.